README.md

```diff
@@ -210,4 +219,4 @@ We appreciate all contributions, large and small, and would like to thank all th
 In addition, we'd like to highlight a few notable contributions:
 
 - The GitHub Authorization Team for implementing and contributing the MySQL datastore
-- The Netflix Authorization Team for [sponsoring and being a design partner for Caveats](https://authzed.com/blog/abac-on-spicedb-enabling-netflix-complex-identity-types)
+- The Netflix Authorization Team for sponsoring and being a design partner for caveats
```

Comment:
Can we add logos or at least favicon size images next to GitHub and Netflix?

Reply:
IDK if it looks like an improvement:

<img width="803" alt="Screenshot 2024-02-18 at 8 53 37 PM" src="https://github.com/authzed/spicedb/assets/343539/80f10961-a402-4324-9082-7ce8ae351b8c">